src/AWS.Deploy.Common/Recipes/Validation/OptionSettingItemValidatorConfig.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.\r
+// SPDX-License-Identifier: Apache-2.0
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace AWS.Deploy.Common.Recipes.Validation
+{
+    public class OptionSettingItemValidatorConfig
+    {
+        [JsonConverter(typeof(StringEnumConverter))]
+        public OptionSettingItemValidatorList ValidatorType {get;set;}
+        public object? Configuration {get;set;}
```

Comment:
can this ever be null? I think this should be a non-nullable type

Reply:
The validator configuration can be null as its object is always created by de-serializing the validator when parsing the recipe. Its perfectly valid for a validator to have no configuration. For example, the `Required` validator or the `FargateTaskSizeCpuMemoryLimits` validator can be specified without any configurations.